minibear/src/main/java/frc/robot/NavX/ContinuousAngleTracker.java

```diff
@@ -0,0 +1,122 @@
+package frc.robot.NavX;
```

Comment:
Who wrote this class?  Need a header to describe if it was ours our ripped off of the internet.